assets/eip-3525/contracts/IERC3525.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+import "./openzeppelin/IERC165.sol";
+import "./openzeppelin/IERC721.sol";
```

Comment:
```suggestion
import "IERC165.sol";
import "IERC721.sol";
```
Finding an implementation for dependencies is left as an exercise for the reader.